product_docs/docs/biganimal/release/overview/03_security/index.mdx

```diff
@@ -25,14 +25,14 @@ All data in BigAnimal is encrypted in motion and at rest. Network traffic is enc
 
 ### Your own encryption key - Transparent Data Encryption (TDE)
 
-Optionally enable Transparent Data Encryption (TDE) at the database level on BigAnimal's cloud account, on AWS, GCP or Azure. TDE encrypts all data files, the write-ahead log (WAL) and temporary files used during query processing and database system operations. 
+Optionally enable Transparent Data Encryption (TDE) at the database level on BigAnimal's cloud account, AWS, GCP, or Azure. TDE encrypts all data files, the write-ahead log (WAL), and temporary files used during query processing and database system operations. 
 
 You can't enable nor disable TDE on existing clusters. To enable TDE, first connect the encryption keys to BigAnimal at the project level, and then select those keys while creating a cluster. 
 
-EDB supports enabling TDE with your own encryption key on Single Node and Primary/Standby High Availability deployments running EDB Postgres Advanced Server or EDB Postgres Extended Server versions 15 and later.
+EDB supports enabling TDE with your own encryption key on single-node and primary/standby high-availability deployments running EDB Postgres Advanced Server or EDB Postgres Extended Server versions 15 and later.
 Both the key and cluster must be in the same region and hosted by the same underlying cloud provider. 
 
-This overview shows the supported cluster-to-key combinations: 
+This overview shows the supported cluster-to-key combinations.
```

Comment:
No colon to introduce tables. (Yes, the release notes uses it, but I haven't asked them to update the template.)